src/deadline/cinema4d_adaptor/Cinema4DClient/cinema4d_handler.py

```diff
@@ -50,6 +50,13 @@ def __init__(self, map_path: Callable[[str], str]) -> None:
         self.take = "Main"
         self.map_path = map_path
 
+    def _remap_assets(self):
+        asset_list = []
+        c4d.documents.GetAllAssetsNew(self.doc, allowDialogs=False, lastPath="", assetList=asset_list)
+        for asset in asset_list:
+            asset_owner = asset.get("owner")
```

Comment:
I gave this code a try but got `None` as the return value for the `.c4d` file's asset owner, leading to an error trying to index into the `None` object on the line below. 

Have you also encountered this error? This could be as simple as adding an `if asset_owner` condition.

If I only apply this owner to `non-None` asset owners, I still appear to get a segmentation fault from https://github.com/aws-deadline/deadline-cloud-for-cinema-4d/issues/118, but it seems like that is out of scope of his change?
```
2024/12/12 15:52:50-06:00 ADAPTOR_OUTPUT: STDERR: /usr/local/lib/python3.9/site-packages/deadline/cinema4d_adaptor/Cinema4DAdaptor/adaptor.sh: line 55: 185665 Segmentation fault      (core dumped) $C4DEXE ${ARGS[*]}
2024/12/12 15:52:50-06:00 openjd_fail: Error encountered while running adaptor: Cinema4D exited early and did not render successfully, please check render logs. Exit code 139
2024/12/12 15:52:50-06:00 ERROR: Entrypoint failed: STDERR: /usr/local/lib/python3.9/site-packages/deadline/cinema4d_adaptor/Cinema4DAdaptor/adaptor.sh: line 55: 185665 Segmentation fault      (core dumped) $C4DEXE ${ARGS[*]}
2024/12/12 15:52:50-06:00 openjd_fail: Error encountered while running adaptor: Cinema4D exited early and did not render successfully, please check render logs. Exit code 139
```

Reply:
Checking into that, thanks!

Reply:
Updated with a guard and now using `paramId` instead of the hardcoded attribute